tests/test_detect.py

```diff
@@ -0,0 +1,33 @@
+from unittest import TestCase
+
+from i18n.detect import Detect
+
+
+class TestExtract(TestCase):
+    """
+    Tests functionality of i18n/detect.py
+    """
+    def test_detect_changes(self):
+        """
+        Asserts that the correct value is returned from detect_changes method
```

Comment:
Verifies the `detect_changes` method can detect changes in translation source files.